index.html

```diff
@@ -38,24 +38,24 @@ <h1>Im Lucca
         Im a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
         </p>
         </div>
-        <p class="let"><img src="images/LETSCONNECT.jpg" alt=""></p>
+        <p class="let"><img src="images/LETSCONNECT.jpg" alt="Hot to Connect with my website"></p>
         <div>
-            <ul class="point2">
-                <li> <a href="https://twitter.com/LuccaMicale_"><img class="twitter" src="images/Vectortw.jpg" alt=""></a></li>
-                <li> <a href="#"><img class="letteri" src="images/VectorI.png" alt=""></a></li>
-                <li> <a href="#"><img class="lettern" src="images/Vectorn.jpg" alt=""></a></li>
-                <li> <a href="#"><img class="letterm" src="images/Vectorm.jpg" alt=""></a></li>
-                <li> <a href="https://github.com/luccamicale"><img class="git" src="images/Vectorgit.jpg" alt=""></a></li>
-                <li> <a href=""><img class="hand" src="images/Vectorhnad.jpg" alt=""></a></li>
+            <ul tabindex="0" class="point2">
+                <li> <a href="https://twitter.com/LuccaMicale_"><img class="twitter" src="images/Vectortw.jpg" alt="Twitter"></a></li>
+                <li> <a href="#" ><img class="letteri"  src="images/VectorI.png" alt="Login in Linkedin"></a></li>
+                <li> <a href="#"><img class="lettern" src="images/Vectorn.jpg" alt="Login in Linkedin"></a></li>
+                <li> <a href="#"><img class="letterm" src="images/Vectorm.jpg" alt="Medium"></a></li>
+                <li> <a href="https://github.com/luccamicale"><img class="git" src="images/Vectorgit.jpg" alt="Enter in my Github"></a></li>
+                <li> <a href=""><img class="hand" src="images/Vectorhnad.jpg" alt="Angellist"></a></li>
             </ul>
         </div>
     </section>
 
     <section id="work">
         <div class="tonic1">
             <div>
-                <img class="imag-tonic" src="images/SnapshootPortfolio(2).jpg" alt="">
-                <img class="imag-tonic1" src="images/Snapshoot22Portfolio.png" alt="">
+                <img class="imag-tonic" src="images/SnapshootPortfolio(2).jpg" alt="image of Tonic">
+                <img class="imag-tonic1" src="images/Snapshoot22Portfolio.png" alt="image of tonic complement">
```

Comment:
- [ ] Again here the description of the image could be clearer like `Project Image` since these images are snapshots of the project you have worked on. Kindly fix this for all the image cards